### DISCUSSION TITLE
What language does the word "sarcasm" come from?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#708] [Galaxy Trivia] [answered]
**Bonus:** What was the literal meaning?

**Comment by peterbaccaro** [comment, marked as answer]
The word "sarcasm" comes from the **Greek** word "sarkasmos," which means "**to tear flesh**" or "**to gnash the teeth in rage**." In its original usage, sarcasm referred to a type of biting or cutting remark intended to hurt or mock someone. The modern usage of the term generally refers to a type of irony or humor that is meant to be humorous or ironic rather than hurtful, although it can still be used in a more negative or critical sense as well.

**Comment by SANKARAMDAS** [comment]
Fittingly, the word sarcasm comes from the ancient Greek word sarkázein, which meant “to tear flesh.” That verb became a metaphor for “speaking bitterly.”Sarcasm is defined as a “sneering or cutting remark.”

**Comment by thuongtruong1009** [comment]
The word "sarcasm" comes from the Greek language. It is derived from the Greek word "sarkasmos," which is related to the verb "sarkazein," meaning "to tear flesh" or "to bite the lips in rage, sneer."

The literal meaning of "sarcasm" in its original Greek context referred to a form of irony or mocking humor that involved tearing or stripping away the flesh. It conveyed the idea of using cutting or biting remarks to mock or ridicule someone or something. Over time, the term "sarcasm" has evolved to represent a form of verbal irony in which remarks are made to convey the opposite of their literal meaning, often with a mocking or contemptuous tone.

**Comment by ParanoidUser** [comment]
**Extra Bonus:** What other English words starting with the letter "M" that come from the Greek language do you know?

**Reply by peterbaccaro** [reply]
1. **Mathematics** - from the Greek "mathema", meaning science or knowledge.
2. **Music** - from the Greek "mousike", meaning art of the muses.
3. **Metaphor** - from the Greek "metaphora", meaning transfer or carry over.
4. **Museum** - from the Greek "mouseion", meaning place of the muses.
5. **Microphone** - from the Greek "mikros" meaning small and "phone" meaning sound.
6. **Megaphone** - from the Greek "megas" meaning large and "phone" meaning sound.
7. **Mythology** - from the Greek "mythos" meaning story or legend.
8. **Monarchy** - from the Greek "monarkhia" meaning rule by one.
9. **Monastery** - from the Greek "monasterion" meaning a place for monks to live and work.
10. **Morphology** - from the Greek "morph" meaning shape and "logos" meaning study of.
11. **Marathon** - from the Greek "marathonas" the name of the town in Greece famous for the battle of Marathon.
12. **Mosaic** - from the Greek "mousaikos" meaning of the muses or artistic.
13. **Meteorology** - from the Greek "meteoron" meaning things of the air and "logos" meaning study of.
14. **Morphine** - from the Greek "Morpheus" the god of dreams.
15. **Mythical** - from the Greek "mythikos" meaning legendary or fictional.

**Comment by SANKARAMDAS** [comment]
Metalinguistics
